modules/ROOT/pages/authentication-authorization/ldap-integration.adoc

```diff
@@ -97,9 +97,24 @@ runtime using xref:reference/procedures.adoc#procedure_dbms_setconfigvalue[`dbms
 
 First, you configure Neo4j to use LDAP as an authentication and authorization provider.
 
-. Uncomment the setting `dbms.security.auth_enabled=false` and change its value to `true` to turn on the security feature.
+. Make sure security is turned on.
+For this purpose uncomment the setting `dbms.security.auth_enabled=false` and change its value to `true`.
 . Uncomment the settings `dbms.security.authentication_providers` and `dbms.security.authorization_providers` and change their value to `ldap`.
 This way, the LDAP connector is used as a security provider for both authentication and authorization.
++
+Note these configuration values are comma-separated lists, so if you wish to continue to use native authentication and authorization alongside LDAP, then these providers can be added to the existing `native` provider.
+The providers will be queried in the order they are declared.
```

Comment:
```suggestion
```

Reply:
Because the sentence above contains the information.

Reply:
The sentence above tells only that you can include different auth providers in the list but it doesn't tell in what oder those providers will be queried: first native, then LDAP or in vice versa. Or the order doesn't mean anything.

Reply:
Ah, I see what you mean. So, if you accept my suggestion:
"If you want, you can still use the `native` provider for mixed-mode authentication and authorization.
The values are comma-separated and queried in the declared order."
It contains the information and  "The providers will be queried in the order they are declared." is not needed.